packages/components/src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -57,8 +89,12 @@ const PreviewSingle = props => {
     );
 };
 
-const FileDropzone = ({ className, noClick = false, ...props }) => {
+const FileDropzone = ({ className, noClick = false, ...props }: TFileDropzone) => {
     const dropzone_ref = React.useRef(null);
+
+    const message = props.message();
+    const validationErrorMessage = props.validation_error_message();
```

Comment:
```suggestion
```
@likhith-deriv i suggest that we continue using props.message and props.validation_error_message the same way as before without turning them into callbacks and without using additional variables returned from them 🙂